lib/wasi/src/bin_factory/exec.rs

```diff
@@ -195,6 +190,124 @@ pub fn spawn_exec_module(
     Ok(join_handle)
 }
 
+/// Finishes with a module and notifies an errcode
```

Comment:
This function seems pretty redundant.

Reply:
The function is gone now since it was reverted.

Reply:
The function has been removed